### PR TITLE
#63 resourece tree to list converter

### LINKED ISSUE #63: Nested resources are not processed
When a template contains nested resources(like EH namespace with an EventHub inside), they won't be displayed.

### PR DESCRIPTION
**Pull request by BienioM**
My proposition to fix #63

### CODE REVIEW

**Comment by kamil-mrzyglod** [comment]
Please run `npm run lint` to fix some code convention errors :) I'll update the readme file to provide some hints regarding contributions.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (kamil-mrzyglod) [inline .vscode/launch.json]: Please add this file to `.gitignore` :)

Comment (BienioM) [reply .vscode/launch.json]: Ok, ready.

**Comment by kamil-mrzyglod** [comment]
Unfortunately it fails two tests:

`
FAIL  test/parsers/templateParser.spec.js
  ? parsers › templateParser › should connect resource and dependency

    TypeError: Cannot read property 'name' of undefined
      
      at Object.it (test/parsers/templateParser.spec.js:127:47)
          at new Promise (<anonymous>)
      at Promise.resolve.then.el (node_modules/p-map/index.js:46:16)
          at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)

 FAIL  test/reducers/fileDialog.spec.js
  ? reducers › fileDialog › should handle OPEN_FILE

    expect(value).toMatchSnapshot()
    
    Received value does not match stored snapshot 1.
    
    - Snapshot
    + Received
    
    @@ -1,11 +1,11 @@
     Object {
       "fileData": Object {
         "characters": 31,
         "contentVersion": undefined,
         "lines": 1,
    -    "loadedIn": 0,
    +    "loadedIn": 1,
         "outputs": Array [],
         "parameters": Array [],
         "resources": Array [],
         "schema": "",
         "variables": Array [],
      
      at Object.it (test/reducers/fileDialog.spec.js:11:205)
          at new Promise (<anonymous>)
      at Promise.resolve.then.el (node_modules/p-map/index.js:46:16)
          at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)
`

while the second one we can mitigate, the first one has to be fixed.

**Pull request merged**